homeassistant/components/zimi/entity.py

```diff
@@ -0,0 +1,67 @@
+"""Base Platform for zimi integrations."""
+
+from __future__ import annotations
+
+import logging
+
+from zcc import ControlPoint
+from zcc.device import ControlPointDevice
+
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity import ToggleEntity
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class ZimiEntity(ToggleEntity):
```

Comment:
```suggestion
class ZimiEntity(Entity):
```
this is a base entity and as such should only derive from entity directly